docs/blog/newsletter-april-2019.rst

```diff
@@ -0,0 +1,114 @@
+
+* Mentor organizations can `apply to participate <https://developers.google.com/season-of-docs/docs/admin-guide>`_ from 2 April to 23 April.
+* Technical writing participants can `apply to participate <https://developers.google.com/season-of-docs/docs/tech-writer-guide>`_ from 29 May to 28 June.
+* The Season of Docs begins on 2 September!
+
+A complete program timeline is available `here <https://developers.google.com/season-of-docs/docs/timeline>`_.
+
+-------------------------------------------------------------
+What's in a code comment? And other musings on developer docs
+-------------------------------------------------------------
+
+One contributor's sharing of a discussion `elsewhere on the web <https://hackaday.com/2019/03/05/good-code-documents-itself-and-other-hilarious-jokes-you-shouldnt-tell-yourself/>`_ about the fallacy of "self-documenting code" produced a lively extension of the conversation into what makes not only good code comments, but good docs more generally. Some highlights:
+
+* No, Virginia, you cannot get all the information you need from inspecting the code only.
+* But there is such a thing as too much information. You don't need to restate the obvious, in code comments or in documenting a GUI.
+* The really difficult task, in documenting code as in any other software interface, is determining what the user needs to know to succeed in their tasks. 
+* We always need to think about different user personas, different needs, and how best to address them all without either hiding information or letting a crucial detail get lost in describing all the other details.
+
+---------------------------------
+Dealing with competing priorities
+---------------------------------
+
+In a discusson on #lone-writer, someone asked: What do you do when lots of people are asking for your time? Particularly when everyone wants to be the highest priority, and you really want to help them all out. It can mean you end up always reacting to the latest emergency, with no time to take a breath and work strategically. How do you avoid burning out like that?
+
+Lots of commenters talked about helping those asking you understand the situation: the different demands on your time, the work you're doing, and the constraints. One way is to make the work visible: for example, having a board so you can show people what's in flight. It can help them understand your priorities: you're fully booked this week, because of another team's work.
+
+It's worth checking if the person requesting can help by gathering the information needed. If they agree to do the groundwork, it takes some of the burden of emergency work off you.
+
+Be aware that you likely won't find time to think strategically: you have to make it. Suggested blocking off time in your calendar, even if you can only start small. For an idea of what to do, see `Neal's talk on triaging </videos/portland/2018/where-do-i-start-the-art-and-practice-of-documentation-triage-neal-kaplan/>`_.
+
```

Comment:
Should the second sentence start with "Documentarians" or "Folks" or something like that? "Folks suggested blocking..."

Reply:
Agreed!